composer.json

```diff
@@ -23,7 +23,8 @@
     "ext-curl": "*",
     "roave/security-advisories": "dev-master",
     "squizlabs/php_codesniffer": "~3.x-dev",
-    "symfony/phpunit-bridge": "~6.0"
+    "symfony/phpunit-bridge": "~6.0",
+    "phpunit/phpunit": "^9.5"
```

Comment:
This change should not be needed as the phpunit-bridge already uses the config from SYMFONY_PHPUNIT_VERSION to choose the correct version.

Reply:
I needed to add `phpunit/phpunit` as a dev dependency, because it `composer install` would not install it without it. I need phpunit installed in den `vendor` folder for my IDE to work properly and more importantly, because otherwise I could not run:

```shell
php ./vendor/bin/phpunit
```

Reply:
It took me a while to remember why you have this issue.  
The thing is that, with symfony's phpunit-bridge, the command to use to run phpunit is not the standard one : you need to run `php ./vendor/bin/simple-phpunit`.  
The command will automaticalley download the correct phpunit version on first run.

Therefore I still do not believe this change is neeeded.